publishing.gradle

```diff
@@ -1,77 +1,103 @@
 apply plugin: 'maven-publish'
-apply plugin: 'com.jfrog.bintray'
+apply plugin: 'signing'
+apply plugin: 'org.jetbrains.dokka'
 
-project.group = project.ext.group
-project.version = project.ext.version
-
-bintray {
-    user = project.hasProperty("BINTRAY_USERNAME") ? project.property('BINTRAY_USERNAME') : null
-    key = project.hasProperty("BINTRAY_API_KEY") ? project.property('BINTRAY_API_KEY') : null
-    pkg {
-        repo = project.ext.name
-        name = getArtifactId()
-        userOrg = 'trademe'
-        licenses = ['Apache-2.0']
-        vcsUrl = project.ext.scm
-        version {
-            name = project.version
-            desc = project.ext.description
-            released = new Date()
-            vcsTag = project.version
-        }
-        publications = ['Bintray']
+task androidSourcesJar(type: Jar) {
+    archiveClassifier.set('sources')
+    if (project.plugins.findPlugin("com.android.library")) {
+        // For Android libraries
+        from android.sourceSets.main.java.srcDirs
+        from android.sourceSets.main.kotlin.srcDirs
+    } else {
+        from sourceSets.main.java.srcDirs
+        from sourceSets.main.kotlin.srcDirs
     }
+
+
+}
+task javadocJar(type: Jar, dependsOn: dokkaJavadoc) {
+    archiveClassifier.set('javadoc')
+    from dokkaJavadoc.outputDirectory
 }
 
-// Create the pom configuration:
-def pomConfig = {
-    licenses {
-        license {
-            name "MIT License"
-            url "http://www.opensource.org/licenses/mit-license.php"
-            distribution "repo"
-        }
-    }
-    scm {
-        url ext.scm
-    }
+artifacts {
+    archives androidSourcesJar
+    archives javadocJar
 }
 
 task sourceJar(type: Jar, dependsOn: 'classes') {
     from sourceSets.main.allSource
     classifier = 'sources'
 }
 
```

Comment:
This task can be removed. It's not used by anything anymore

Reply:
removed unused task